internal/pkg/cli/pipeline_deploy_test.go

```diff
@@ -164,7 +167,7 @@ stages:
 					m.prog.EXPECT().Stop(log.Ssuccessf(fmtPipelineDeployResourcesComplete, appName)).Times(1),
 
 					m.ws.EXPECT().ReadPipelineManifest().Return([]byte(content), nil),
-					m.ws.EXPECT().ListWorkloads().Return([]string{"frontend", "backend"}, nil).Times(1),
+					m.actionCmd.EXPECT().Execute().Times(2),
```

Comment:
Once for service listing and once for job listing right?

Reply:
yeah